api/src/main/java/com/datastrato/gravitino/rel/expressions/partitions/Partitions.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.rel.expressions.partitions;
+
+import com.datastrato.gravitino.rel.expressions.NamedReference;
+import com.datastrato.gravitino.rel.expressions.literals.Literal;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Objects;
+
+/** The helper class for partition expressions. */
```

Comment:
please provide more descriptions about IdentityPartition, rangePartition and listPartition, I'm confused

Reply:
More comments are added to the static method, and UTs can also be referred to